common/static/sass/edx-pattern-library-shims/base/_variables.scss

```diff
@@ -178,6 +178,22 @@ $btn-brand-active-background:             $uxpl-blue-base !default;
 $btn-brand-disabled-background:           #f2f3f3 !default;
 $btn-brand-disabled-color:                #676666 !default;
 
+// Upgrade button
+$btn-upgrade-border-color:                $uxpl-green-base !default;
```

Comment:
@staubina Did someone suggest that you put these "Upgrade button" specific styles in the general pattern library him file? It seems like a very odd location to me-- I would expect them to be in the sass file that contains things specific to the course info page.